model/src/main/java/com/epam/repair/model/Executor.java

```diff
@@ -28,21 +28,21 @@
      */
     @ManyToOne
     @JoinColumn(name="first_name_id")
-    private FirstName firstName;
+    private FirstNameDTO firstName;
```

Comment:
Here can't be `dto` object because it's DB view

Reply:
Comment accepted. Fixed in the branch mapstruct-model.